integration_tests/settlement_market.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Settlement Market Tests
+
+Scenario: Assets can post bids and offers
+  Given redis container is started
+  And gsy-e is started in paused mode using setup strategy_tests.external_devices_settlement_market (-t 60s -s 60m -d 4h --slot-length-realtime 2s)
+  When the gsy-e-sdk is connecting to gsy-e with test_aggregator_settlement
```

Comment:
Where did you implement this step?

Reply:
I forgot to commit this file initially. Done here [f9271ad](https://github.com/gridsingularity/gsy-e-sdk/pull/207/commits/f9271ad5ac9928e039b368fba0e240b78a3a03a6).